e2e/cypress/integration/api/iam/14_projects_delete_api.spec.ts

```diff
@@ -0,0 +1,146 @@
+import { describeIfIAMV2p1 } from '../../../support/constants';
+import { uuidv4 } from '../../../support/helpers';
+
+describeIfIAMV2p1('Project delete', () => {
+  const cypressPrefix = 'test-project-delete';
+
+  const project =  {
+    id: `${cypressPrefix}-project-delete-applied-rule${Cypress.moment().format('MMDDYYhhmm')}`,
+    name: 'project delete'
+  };
+
+  const rule = {
+    id: 'rule-applied',
+    name: 'rule',
+    type: 'NODE',
+    project_id: project.id,
+    conditions: [
+      {
+        attribute: 'CHEF_SERVER',
+        operator: 'EQUALS',
+        values: ['example.org']
+      }
+    ]
+  };
+
+  before(() => {
+    cy.cleanupV2IAMObjectsByIDPrefixes(cypressPrefix, ['projects', 'policies']);
+  });
+
+  after(() => cy.cleanupV2IAMObjectsByIDPrefixes(cypressPrefix, ['projects', 'policies']));
+
+  it('deleting a project', () => {
+    // Create a project
+    cy.request({
+      headers: { 'api-token': Cypress.env('ADMIN_TOKEN') },
+      method: 'POST',
+      url: '/apis/iam/v2beta/projects',
+      body: project
+    });
+
+    // Add a rule to the project
+    cy.request({
+      headers: { 'api-token': Cypress.env('ADMIN_TOKEN') },
+      method: 'POST',
+      url: `/apis/iam/v2beta/projects/${project.id}/rules`,
+      body: rule
+    });
+
+    // Try to delete the project with a stagged rule
+    cy.request({
+      headers: { 'api-token': Cypress.env('ADMIN_TOKEN') },
+      method: 'DELETE',
+      url: `/apis/iam/v2beta/projects/${project.id}`,
+      failOnStatusCode: false
+    }).then((deleteResp) => {
+      expect(deleteResp.status,
+        'Did not fail deleting a project with a stagged rule').to.be.oneOf([400]);
```

Comment:
s/stagged/staged

Reply:
Also I think we could simplify the assertion to `expect(deleteResp.status).to.equal(400);`